src/pages/moderator/paddle/adjustments.tsx

```diff
@@ -0,0 +1,210 @@
+import {
+  Anchor,
+  Badge,
+  Button,
+  Center,
+  Container,
+  Group,
+  Loader,
+  Paper,
+  Select,
+  Stack,
+  Table,
+  Text,
+  TextInput,
+  Title,
+} from '@mantine/core';
+import { useDebouncedValue } from '@mantine/hooks';
+import { useState } from 'react';
+import { NotFound } from '~/components/AppLayout/NotFound';
+import { usePaddleAdjustmentsInfinite } from '~/components/Paddle/util';
+import { useFeatureFlags } from '~/providers/FeatureFlagsProvider';
+import { usePaddle } from '~/providers/PaddleProvider';
+
+import { AdjustmentAction, GetPaddleAdjustmentsSchema } from '~/server/schema/paddle.schema';
+import { formatDate } from '~/utils/date-helpers';
+import { formatPriceForDisplay } from '~/utils/number-helpers';
+import { getDisplayName, toPascalCase } from '~/utils/string-helpers';
+
+export default function ModeratorPaddleAdjustments() {
+  const [filters, setFilters] = useState<GetPaddleAdjustmentsSchema>({
+    limit: 50,
+    customerId: [],
+    subscriptionId: [],
+  });
+  const { paddle } = usePaddle();
+  const [debouncedFilters, cancel] = useDebouncedValue(filters, 500);
+  const { adjustments, isLoading, isFetching, fetchNextPage, hasNextPage } =
+    usePaddleAdjustmentsInfinite(debouncedFilters);
+  const featureFlags = useFeatureFlags();
+
+  if (!featureFlags.paddleAdjustments) {
+    return <NotFound />;
+  }
+
+  return (
+    <Container size="lg">
```

Comment:
Just to be on the err side (cause I don't remember if we do this for all `/moderator/*` routes already). Let's add the `<Meta deIndex />` component at the top here
```suggestion
  return (
    <>
      <Meta title="Paddle Adjustments | Moderator" deIndex />
      <Container size="lg">
```